src/components/Reports/ProjectReport/PiechartByProject/PieChartByProject.css

```diff
@@ -0,0 +1,20 @@
+
+.pie-chart-title{
+  text-align: left;
+  margin-bottom: 24px;
+
+}
+.pie-chart-description{
```

Comment:
The user name seems to be chipped off for bigger screen sizes. 

![PR 2310](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/59435391/8915aca2-3d2c-4276-8e96-9b345c8dac59)

Making the text size relative to the circle would be a good start.

Reply:
Hello,
Thanks for your review, 
Please check it again